client/electron/build_main.action.mjs

```diff
@@ -17,13 +17,13 @@ import path from 'path';
 import url from 'url';
 
 import electronMainWebpackConfigs from './webpack_electron_main.mjs';
-import {getBuildParameters} from '../../client/src/build/get_build_parameters.mjs';
-import {getWebpackBuildMode} from '../../client/src/build/get_webpack_build_mode.mjs';
-import {runWebpack} from '../../client/src/build/run_webpack.mjs';
-import {getRootDir} from '../build/get_root_dir.mjs';
-import {runAction} from '../build/run_action.mjs';
+import {getRootDir} from '../../src/build/get_root_dir.mjs';
+import {runAction} from '../../src/build/run_action.mjs';
+import {getBuildParameters} from '../src/build/get_build_parameters.mjs';
```

Comment:
Not for this PR, but I believe get_build_parameters is specific to the client app, so it should probably move into `client/`